install_config/oab_broker_configuration.adoc

```diff
@@ -740,3 +740,92 @@ secrets:
+been configured.
+
+[[configuring-oab-proxy-prerequisites]]
+=== Prerequisites
+
+Some important prerequisites necessary for running the broker behind a proxy:
+
+[[configuring-oab-proxy-cluster-proxy]]
+==== Configuring the Cluster to Run Behind a Proxy
+
+Before attempting to run the OAB behind a proxy, review
+xref:../install_config/http_proxies.adoc#install-config-http-proxies[Working
+with HTTP Proxies] and ensure your cluster is configured accordingly to run
+behind a proxy.
+
+In particular, the cluster must be configured to _not_ proxy internal cluster
+requests. This is typically configured with a `NO_PROXY` setting of
+`.cluster.local,.svc`, in addition to any other desired `NO_PROXY` settings.
+This is because the broker must be able to directly communicate with its etcd
+instance.
+
+[NOTE]
+====
+Brokers deploying unversioned, or v1 APBs _must_ also add `172.30.0.1` to their
+`NO_PROXY` list. APBs prior to v2 extracted their credentials from running APB
+pods via an `exec` HTTP request, rather than a secret exchange. Unless you are
+running a broker with experimental proxy support in a cluster prior to
+{product-title} 3.9, you probably do not have to worry about this.
+====
+
+[[configuring-oab-proxy-adapter-whitelists]]
+==== Registry Adapter Whitelists
+
+The broker's configured registry adapters must be able to communicate with their
+external registries in order to bootstrap successfully and load remote APB
+manifests. These requests can be made via the proxy, however, the proxy must
+ensure that the required remote hosts are accessible.
+
+Example required whitelisted hosts:
+
+[options="header"]
+|===
+|Registry Adapter Type |Whitelisted Hosts
+
+|`rhcc`
+|`registry.access.redhat.com`, `access.redhat.com`
+
+|`dockerhub`
+|`docker.io`
+|===
+
+[[configuring-oab-proxy-use-a-proxy]]
+=== Configuring the Broker to Use a Proxy
+
+To configure the broker for external access via proxy, the cluster administrator
+must edit the broker's `DeploymentConfig` and set the following environment
+variables:
+
+- `HTTP_PROXY`
+- `HTTPS_PROXY`
+- `NO_PROXY`
+
+See
+xref:../install_config/http_proxies.adoc#setting-environment-variables-in-pods[Setting Proxy Environment Variables in Pods] for details.
+
+[[configuring-oab-proxy-pods]]
+=== Setting Proxy Environment Variables in Pods
+
+It is common that APB pods themselves may require external access via proxy as
+well. If the broker recognizes it has a proxy configuration, it will
+transparently apply these environment vars to the APB pods that it spawns. As
+long as the modules used within the APB respect proxy configuration via
+environment variable, the APB will also use these settings to perform its work. 
+
+Finally, it is possible the services spawned by the APB may also require
+external network access via proxy. The APB _must_ be authored to set these
+environment variables explicitly if recognizes them in its own execution
```

Comment:
@eriknelson This bit "explicitly if recognizes them" seems like a typo, but I wasn't sure what it should be. Maybe "explicitly if it recognizes them"?